content/en/storage-provider/boost/configuration/overview/index.md

```diff
@@ -571,7 +571,7 @@ SectorIndexApiInfo = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJBbGxvdyI6WyJyZWFkI
 This guide covers all the configuration in use by boostd process. Some of the configuration parameters found in the config.toml file are not used in boost and thus are not covered here. These configuration parameters can be ignored
 {{</alert>}}
 
-<!--STEF I don't know how to force the tables following this comment to wrap reasonably-->
+<!-- STEF JOHNNY I don't know how to force the tables following this comment to wrap reasonably-->
```

Comment:
You can align table content to be left, right, or centre using:

```
| :left-align | :centre-align: | right-align: |
| --- | --- | --- |
| abc | efg | hij |
```

There isn't a way to specify table wrapping with markdown alone, as far as I know. This is a style change we'd have to make ourselves in CSS.

Reply:
I did some experimentation- I couldn't work out why it was wrapping some of the very long strings but not others. Looks like it's waiting for a '-' - which makes sense. I could truncate the very long keys with ellipses, something like : "eyJh...J9.eyJB...XX0.nbSv...E10:/ip4/0.0.0.0/tcp/2345/http - which would make things a bit more manageable? I don't think the full strings have much practical usage. What do you suggest here?

Reply:
image of the problem might be helpful 
<img width="1206" alt="image" src="https://user-images.githubusercontent.com/942675/213747244-65035caa-9432-4e4a-b346-a83c23a78b35.png">

Reply:
Ah yeah, that's a bit of an issue. I don't have much time to fix this right now though -- focused pretty heavily on FVM stuff. Can you make an issue, if you haven't already?

Reply:
Issue created: https://github.com/filecoin-project/filecoin-docs/issues/1611